st2client/st2client/commands/action_alias.py

```diff
@@ -127,3 +132,89 @@ def run_and_print(self, args, **kwargs):
         print("Matching Action-alias: '%s'" % execution.actionalias['ref'])
         print("To get the results, execute:\n st2 execution get %s" %
               (execution.execution['id']))
+
+
+class ActionAliasTestCommand(ActionRunCommandMixin, resource.ResourceCommand):
+    display_attributes = ['name']
+
+    def __init__(self, resource, *args, **kwargs):
+        super(ActionAliasTestCommand, self).__init__(
```

Comment:
Since we only care about Python 3 now, this can just be:
```suggestion
        super().__init__(
```

Reply:
I used previous approach for consistency reasons, but if we are fine using this notation everywhere (and potentially updating existing code), I'm also fine with it.